src/extension/runner/index.ts

```diff
@@ -279,13 +282,15 @@ export default class GrpcRunner implements IRunner {
 
     const id = runnerEnv.getSessionId()
 
-    const { session } = await this.client.getSession(GetSessionRequest.create({ id })).response
+    const { session } = await this.client.getSession(GetSessionRequestImpl().create({ id }))
+      .response
 
     if (!session) {
       return undefined
     }
 
-    convertEnvList(session.envs)
+    const env = (session as any).envs ?? (session as any).env ?? []
```

Comment:
It looks redundant, but after a review of other alternatives, I think the current one makes sense. :+1: